src/modules/sync/lib/convert-tree-to-options-to-sync.ts

```diff
@@ -49,7 +49,10 @@ export const convertTreeToOptionsToSync = ( tree: TreeNode[] ): SyncOptionsWithS
 			) {
 				const selectedItems = item.children
 					.filter( ( child ) => child.checked )
-					.map( ( child ) => child.id );
+					.map( ( child ) => {
+						const prefix = `${ item.id }-`;
+						return child.id.startsWith( prefix ) ? child.id.slice( prefix.length ) : child.id;
+					} );
```

Comment:
The last one thing - I was thinking, how to avoid this code and - WDYT about adding property name to the items of the tree?
So we have:
1. `id` what should be uniqueue and is used for updating tree items
2. `name` which is original items id (e.g. `index.php` or `akismet`)
As a result - we don't mix logic and can keep everything clear and separated.

```suggestion
.map( ( child ) => child.name );
```